[libxft] Unable to build

```
/bin/grep: /usr/local/lib/libfreetype.la: No such file or directory
/bin/sed: can't read /usr/local/lib/libfreetype.la: No such file or directory
libtool: link: `/usr/local/lib/libfreetype.la' is not a valid libtool archive
make[2]: *** [Makefile:442: libXft.la] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/libXft-2.3.2.tar.gz.dir/libXft-2.3.2/src'
make[1]: *** [Makefile:493: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/libXft-2.3.2.tar.gz.dir/libXft-2.3.2'
make: *** [Makefile:393: all] Error 2
libxft failed to build: `make V=0 -j2` exited with 2
```

--- a/packages/libxft.rb
+++ b/packages/libxft.rb
@@ -10,6 +10,8 @@ class Libxft < Package
   depends_on 'libxrender'
   depends_on 'libx11' 
   depends_on 'fontconfig'
+  depends_on 'util_macros'
+  depends_on 'zlibpkg'
   
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"

